test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs

```diff
@@ -599,6 +599,11 @@ protected static void ExecuteApplication(string path, string args, out string st
             throw new ArgumentException("Executable path must not be null or whitespace.", nameof(path));
         }
 
+        if (!File.Exists(path))
```

Comment:
Without this check we end up with error like `System.ComponentModel.Win32Exception: No such file or directory` which isn't really helpful.